public/javascripts/payments.js

```diff
@@ -283,6 +313,20 @@
     submitButton.disabled = true;
     submitButton.textContent = 'Processing…';
 
+    // Update Payment Intent if currency is different to default
+    if(config.currency !== activeCurrency)
+    {
```

Comment:
Might want to neaten this up by removing the newline and adding a space after `if`.

Reply:
Thanks! Updated this and another line with the same missing space.